invenio_rdm_migrator/streams/files/transform.py

```diff
@@ -111,7 +111,6 @@ def transform(self, entry):
 class FilesObjectVersionEntry(Entry):
     """Transform a single file object version entry."""
 
-    @abstractmethod
     def _version_id(self, entry):
```

Comment:
I would keep the @abstractmethod around and just return None in zenodo-rdm implementation with a comment that these are auto-generated or passed from community metadata...